Formula/cryfs.rb

```diff
@@ -0,0 +1,32 @@
+require "open3"
+
+class Cryfs < Formula
+  desc "Encrypts your files, so you can safely store them in Dropbox, iCloud, ..."
+  homepage "https://www.cryfs.org"
+  url "https://github.com/cryfs/cryfs.git", tag: "0.9.7"
+  head "https://github.com/cryfs/cryfs.git", branch: "develop"
+
+  depends_on "cmake" => :build
+  depends_on :python => :build
+  depends_on "openssl"
+  depends_on "boost"
+  depends_on "cryptopp"
+  depends_on :osxfuse
+
+  needs :cxx11
+
+  def install
+    mkdir("build") do
+      #TODO Homebrew is passing in CXXFLAGS to cmake which disable -O3. Can it be built with -O3?
```

Comment:
It can, but we prefer the default Homebrew `-Os` unless there's something that breaks with it.